plugins/admin_console/system_partner/lib/api/KalturaSystemPartnerLimitArray.php

```diff
@@ -15,11 +15,20 @@ public static function fromPartner(Partner $partner)
 		$reflector = KalturaTypeReflectorCacher::get('KalturaSystemPartnerLimitType');
 		$types = $reflector->getConstants();
 		foreach($types as $typeInfo) {
-		    $typeValue = $typeInfo->getDefaultValue();
-		    $arr[] = KalturaSystemPartnerOveragedLimit::fromPartner($typeValue, $partner);
+			$typeValue = $typeInfo->getDefaultValue();
+			if ($typeValue == KalturaSystemPartnerLimitType::LIVE_CONCURRENT_BY_ADMIN_TAG)
+			{
+				$adminTagLimits =  KalturaSystemPartnerLiveAdminTagLimit::getArrayFromPartner($partner);
+				foreach ($adminTagLimits as $adminTagLimit)
+				{
+					$arr[] = $adminTagLimit;
+				}
+			} 
+			else
+			{
+				$arr[] = KalturaSystemPartnerOveragedLimit::fromPartner($typeValue, $partner);
```

Comment:
We can add function "getArrayFromPartner" on KalturaSystemPartnerOveragedLimit which wrap "fromPartner" and just insert to array - in this case the signature will be the same and the "KalturaSystemPartnerLimit" should have getArrayFromPartner as well and call the sub file according the type